tasks/build.yml

```diff
@@ -37,11 +37,16 @@
     name: "openjdk-{{ sansible_java_version }}-jdk"
     state: present
 
+- name: Get DEB architecture
+  command: dpkg --print-architecture
+  register: deb_architecture
```

Comment:
Can you rename this to `sansible_java_ deb_architecture`, please?  That's to keep the namespace clean.

Reply:
oh thanks for feedback, i have been rename it :D